### DISCUSSION TITLE
Lidl FB20-002

### DISCUSSION THREAD

**Discussion by sohnvonoff** [Koenkk/zigbee2mqtt#8658] [Devices]
I have a Lidl FB20-002 remote switch. When I run this with deconz I bokmme numeric data (1002,2002, ..) as payload. I assume it's the raw data. 
If I run this with zigbee2mqtt I get a character string (brightnes_step_down, ..) as the payload.
Is there a possibility under zigbee2mqtt to output the numerical outputs (raw data) of the FB20-002 as payload data?
Why do i ask? 
The FB20-002 has more codes on the second key (ON), which can be called up by pressing it repeatedly.
These codes are very good for controlling color temperature or scenes. 
These can be used with the numerical output under deconz (after OFF->1002 then ->1004 ->1005 ->1006).
Under zigbee2mqtt the first pressure "ON" is output to the second pressure of the ON button, nothing is more output. OFF must first be pressed again. 
Using DECONZ is not an option as some of my Zigbe devices are not supported. 
Thanks in advance
Bernd

**Comment by SnakeZi** [comment]
When I posted I didn't notice that was a closed thread. But maybe my observations could help with this issue
https://github.com/Koenkk/zigbee2mqtt/issues/12772#issuecomment-1229219610

**Comment by sohnvonoff** [comment]
Hi SnakeZi,
Thank you very much for your hint.
